smartsim/_core/entrypoints/colocated.py

```diff
@@ -177,7 +177,8 @@ def main(
     db_scripts: t.List[t.List[str]],
     db_identifier: str,
 ) -> None:
-    global DBPID  # pylint: disable=global-statement
+    # pylint: disable=too-many-statements
+    global DBPID
```

Comment:
I guess the `pylint` directive was accidentally deleted, but it is needed to make it accept `global` variables.

Reply:
yes oops!